lib/matplotlib/backends/backend_pdf.py

```diff
@@ -550,7 +550,7 @@ def close(self):
                                for val in six.itervalues(self.alphaStates)]))
         self.writeHatches()
         self.writeGouraudTriangles()
-        xobjects = dict(six.itervalues(self.images))
+        xobjects = dict(x[1:] for x in six.itervalues(self.images))
```

Comment:
Then let's take the opportunity and make it private now. It will make breakages very obvious.